src/mainwindow.cpp

```diff
@@ -505,7 +505,7 @@ void MainWindow::setupKeyboardShortcuts()
     new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_Backslash), this, SLOT(resetBlockFormat()));
 
     QxtGlobalShortcut *shortcut = new QxtGlobalShortcut(this);
-#if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
+#if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC)) || defined(Q_OS_WIN)
```

Comment:
At this point is seems easier to reverse this check, and just make `Qt::META | Qt::Key_N` a special case for macOS.

Reply:
Good point.